.pre-commit-config.yaml

```diff
@@ -14,6 +14,10 @@ repos:
     -   id: check-yaml
     -   id: check-merge-conflict
     -   id: check-symlinks
+    -   id: check-executables-have-shebangs
+        exclude: '^(tinygrad_repo)'
```

Comment:
let's make this a global exclude up on top (and only tinygrad for now)

Reply:
done